File: pinyougousellergoodsinterface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.pojo.TbBrand;
import entity.PageResult;

import java.util.List;

public interface BrandService {

    public List<TbBrand> findAll();

        /*
        * 品牌分页
        * pageNum:当前页码
        * pageSize：每页记录数
        * **/
    public PageResult findPage(int pageNum, int pageSize);



    /**
     * 增加
     * */
    public void add(TbBrand brand);


    /*
    * 查看
    * */
    public TbBrand findOne(Long id);

    /*
    * 修改
    * */
    public void update(TbBrand brand);

    /*
    * 删除
    * */
    public void delete(Long [] ids);

    /*
    * 模糊查询
    * */
    public PageResult findPage(TbBrand brand,int pageNum, int pageSize);
}

File: pinyougoumanager/src/main/webapp/js/service/brandService.js
app.service("brandService",function ($http) {
    //查询品牌列表
    this.findAll=function () {
        return $http.get('../brand/findAll.do');
    }

    //增加
    this.add=function (entity) {
        return $http.post('../brand/add.do',entity)
    }
    //修改
    this.update=function (entity) {
        return $http.post('../brand/update.do',entity)
    }
    //查询实体
    this.findOne=function (id) {
        return $http.get('../brand/findOne.do?id='+id);
    }
    //删除
    this.dele=function (ids) {
        return $http.get('../brand/delete.do?ids='+ids);
    }
    //分页
    this.findPage=function (page,size) {
        return  $http.get('../brand/findPage.do?page='+page +'&size='+size);
    }
    //条件查询
    this.search=function (page,size,searchEntity) {
        return $http.post('../brand/search.do?page='+page +'&size='+size, searchEntity);
    }


});
